package.json

```diff
@@ -40,10 +44,12 @@
     "react-native-floating-action": "^1.19.1",
     "react-native-gesture-handler": "^1.5.6",
     "react-native-image-crop-picker": "^0.26.2",
+    "react-native-image-picker": "^2.3.1",
```

Comment:
I suggest to use `react-native-image-crop-picker` as it's already on the dependency list